test/microapps-ui-xcm/microapps-ui-main.accounts.test.ts

```diff
@@ -0,0 +1,300 @@
+
+      const walletWrapper = new WalletWrapper(driver);
+      await walletWrapper.openWalletConnectionInfo();
+      await walletWrapper.openDeposit();
+      const depositModal = new DepositModal(driver);
+      const isModalVisible = await depositModal.isModalVisible();
+      expect(isModalVisible).toBeTruthy();
+
+      await depositModal.openChainList();
+      await depositModal.selectChain("Kusama");
+      await depositModal.openTokensList();
+      await depositModal.waitForTokenListElementsVisible(KSM_ASSET_NAME);
+      await depositModal.selectToken(KSM_ASSET_NAME);
+      await depositModal.enterValue("1");
+
+      await depositModal.waitForContinueState(true, 60000);
+      const isOriginFeeDisplayed = await depositModal.isOriginFeeDisplayed();
+      expect(isOriginFeeDisplayed).toBeTruthy();
+      const isDestinationFeeDisplayed =
+        await depositModal.isDestinationFeeDisplayed();
+      expect(isDestinationFeeDisplayed).toBeTruthy();
+
+      const isContinueButtonEnabled =
+        await depositModal.isContinueButtonEnabled();
+      expect(isContinueButtonEnabled).toBeTruthy();
+
+      await depositModal.clickContinue();
+      await waitForMicroappsActionNotification(
+        driver,
+        mangata,
+        kusama,
+        TransactionType.Deposit,
+        2,
+      );
+    });
+
+    test("Withdraw tokens", async () => {
+      await setupPageWithState(driver, acc_name);
+
+      const walletWrapper = new WalletWrapper(driver);
+      await walletWrapper.openWalletConnectionInfo();
+      await walletWrapper.openWithdraw();
+      const withdrawModal = new WithdrawModal(driver);
+      const isModalVisible = await withdrawModal.isModalVisible();
+      expect(isModalVisible).toBeTruthy();
+
+      await withdrawModal.openChainList();
+      await withdrawModal.selectChain("Kusama");
+      await withdrawModal.openTokensList();
+      await withdrawModal.waitForTokenListElementsVisible(KSM_ASSET_NAME);
+      await withdrawModal.selectToken(KSM_ASSET_NAME);
+      await withdrawModal.enterValue("1");
+
+      await withdrawModal.waitForContinueState(true, 60000);
+      const isOriginFeeDisplayed = await withdrawModal.isOriginFeeDisplayed();
+      expect(isOriginFeeDisplayed).toBeTruthy();
+      const isDestinationFeeDisplayed =
+        await withdrawModal.isDestinationFeeDisplayed();
+      expect(isDestinationFeeDisplayed).toBeTruthy();
+
+      const isContinueButtonEnabled =
+        await withdrawModal.isContinueButtonEnabled();
+      expect(isContinueButtonEnabled).toBeTruthy();
+
+      await withdrawModal.clickContinue();
+
+      const modal = new NotificationModal(driver);
+      await modal.waitForModalState(
+        ModalType.Confirm,
+        TransactionType.Withdraw,
+        3000,
+      );
+      const isModalWaitingForSignVisible = await modal.isModalVisible(
+        ModalType.Confirm,
+        TransactionType.Withdraw,
+      );
+      expect(isModalWaitingForSignVisible).toBeTruthy();
```

Comment:
is there any impediment that does not let us run the deposit itself? Otherwise, I do not understand why the test ends here.